include/core/JamomaSample.h

```diff
@@ -90,6 +91,15 @@ namespace Jamoma {
 		{
 			return mFrameCount;
 		}
+        
+        
+        /**	Return the number of frames dedicated to padding.
```

Comment:
Is this true @nwolek ?  My understanding is that the padding size is half the number of frames dedicated to padding because for each sample of padding there are two frames added (one at the head and one at the tail).